docs/guides/4-custom-rulesets.md

```diff
@@ -86,7 +86,7 @@ responses:
     foo: bar
 ```
 
-## Extending Rules
+## Extending Ruleset
 
 When extending another ruleset, you can actually extend and modify rules it has declared by adding a rule to your own ruleset with the same name.
```

Comment:
And since undeclared properties in the new rule won't inherit from properties declared in the base rule, maybe

```suggestion
When extending another ruleset, you can actually replace a rule it has declared by adding a new rule to your own ruleset with the same name.
```